.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint-config-digitalbazaar",
+};
```

Comment:
Again, do we want 2 spaces here like the rest of our JS?

Reply:
I can quickly change this from 4 spaces to 2 no issue.